x-pack/plugins/security/public/authentication/login/login_page.tsx

```diff
@@ -69,7 +75,7 @@ const loginFormMessages: Record<LogoutReason, NonNullable<LoginFormProps['messag
   },
   LOGGED_OUT: {
     type: LoginFormMessageType.Info,
-    content: i18n.translate('xpack.security.login.loggedOutDescription', {
+    content: i18n.translate('xpack.security.login.loggedOutDescriptionDefault', {
```

Comment:
__nit:__ I guess we can revert this change now (and in l10n files as well)?
```suggestion
    content: i18n.translate('xpack.security.login.loggedOutDescription', {
```